app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -29,6 +29,7 @@ def show
             param :name, String, :required => true
             param :value, String, :required => true
           end
+          param :effective_user, String, :required => false
```

Comment:
>  What about nesting this into under 'ssh' key?

I like that. It keeps the provider-specific settings grouped together and makes it clear what goes with what.